.github/dependabot.yml

```diff
@@ -19,9 +28,6 @@ updates:
     schedule:
       interval: weekly
       day: sunday
-    ignore:
-      - dependency-name: "github.com/open-telemetry/opentelemetry-collector-contrib/*"
-      - dependency-name: "go.opentelemetry.io/collector*"
```

Comment:
These were removed because they were not generated by `dbotconf`. I believe the spirit of these ignores would prevent extra noise from these PRs. I think it's acceptable to let dependabot create these PRs now. It will act as notice to that the dependencies need to be updated. Dependabot go updates are less painful now that we have the dependabot script.